Test/Unit/Model/Signup/RequestTest.php

```diff
@@ -152,7 +152,8 @@ private function getDefaultParams()
             'store_name' => 'Store Name',
             'region' => 1,
             'url' => 'http://www.google.com/',
-            'store_id' => 1
+            'store_id' => 1,
+            'phone' => '123456789'
```

Comment:
Do we not want this let blank?

Reply:
this is just a unit test, testing it gets passed correctly